ci_framework/roles/devscripts/tasks/sub_tasks/16_user.yml

```diff
@@ -14,11 +14,9 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Create the specified user.
-  become: true
-  ansible.builtin.user:
-    name: "{{ cifmw_devscripts_user }}"
-    state: present
+- name: Gather the user
+  ansible.builtin.set_fact:
+    cifmw_devscripts_user: "{{ ansible_user_id }}"
```

Comment:
Do we really need this fact? We can use ansible_user_id straightforward

Reply:
there's also a non-null risk of ppl overriding this via a parameter (while using an "old" parameter file for instance). Might be better to drop it for good.